src/components/auth/registerUser.js

```diff
@@ -0,0 +1,228 @@
+import React, { Component } from "react";
+import Input from "../common/input";
+import Button from "../common/button";
+import "./../../assets/authAssets/test.css";
+import "bootstrap/dist/css/bootstrap.min.css";
+import { connect } from "react-redux";
+
+import { signUpAction } from "../../actions/authActions/signUpAction";
+
+export class RegisterUser extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      newUser: {
+        username: "",
+        email: "",
+        password: "",
+        passwordConfirm: ""
+      },
+      isValidated: false,
+      errors:{}
+    };
+    this.handleFormSubmit = this.handleFormSubmit.bind(this);
+    this.handleInputChange = this.handleInputChange.bind(this);
+  }
```

Comment:
Use arrow functions instead of binding 'this'

Reply:
I have implemented the arrow functions